src/Padawan/Domain/Completer/GlobalFunctionsCompleter.php

```diff
@@ -44,13 +44,28 @@ public function getEntries(Project $project, Context $context)
 
     public function canHandle(Project $project, Context $context)
     {
-        $postfix = "";
-        if (is_string($context->getData())) {
-            $postfix = trim($context->getData());
-        }
+        $postfix = $this->getPostfix($context);
         return parent::canHandle($project, $context)
             && ($context->isString() || $context->isEmpty())
             && strlen($postfix) > 0
             ;
     }
+
+    private function getPostfix(Context $context)
+    {
+        if (is_string($context->getData())) {
+            return trim($context->getData());
+        }
+        if (empty($postfix)) {
+            $contextData = $context->getData();
+            if (is_array($contextData) && @$contextData[3] instanceof \PhpParser\Node\Arg) {
```

Comment:
can this somehow be moved to a Context class or some subclass? `InFunctionCallContext`? It would be cool not to depend on phpparser anywhere except `Parser` namespace
```php
<?php
...
if ($context instanceof InFunctionCallContext && $context->isClassName()) {
  $postfix = $context->postfix();
}